src/main/java/fi/istrange/traveler/resources/AuthResource.java

```diff
@@ -36,22 +37,7 @@ public DefaultJwtCookiePrincipal login(
             @NotNull UserCredentialsView userCredentialsView,
             @Context DSLContext database
     ) {
-        if (userCredentialsView.getName() == null || userCredentialsView.getPassword() == null) {
-            throw new WebApplicationException(422);
-        }
-
-        Optional<Record> result = database.select()
-                .from(Tables.USER_CREDENTIALS)
-                .where(Tables.USER_CREDENTIALS.USERNAME.equal(userCredentialsView.getName()))
-                .fetchOptional();
-
-        if (
-                !result.isPresent() ||
-                !userCredentialsView.getPassword().equals(result.get().getValue(Tables.USER_CREDENTIALS.PASSWORD)) ||
-                !result.get().getValue(Tables.USER_CREDENTIALS.ACTIVE)
-        ) {
-            throw new NotAuthorizedException("Invalid credentials");
-        }
+        auth.authenticate(userCredentialsView.getName(), userCredentialsView.getPassword(), database);
```

Comment:
I pulled the PR and tested it, the authentication behaves the way I expected. As long as the body of the request is ok, it gives me a cookie, not matter what the values for the password and username are.